changelog/6332.feature.deprecation.rst

```diff
@@ -0,0 +1,4 @@
+Moved :func:`~sunpy.visualization.limb.draw_limb` into `~sunpy.visualization.drawing`
+as :func"`~sunpy.visualization.drawing.limb` and added the plot_equator_prime_meridian.py
+example to the Example Gallery.
+Deprecated the `~sunpy.visualization.limb` module.
```

Comment:
This will need a separate changelog per item.

Reply:
You should be able to link the example like you did in the what's new section.

Reply:
Okay, would the moving/renaming of `draw_limb` be considered breaking, or just part of the deprecation?